src/main/java/com/codecool/virtualpub/data/Bartender.java

```diff
@@ -1,40 +1,35 @@
 package com.codecool.virtualpub.data;
 
 import com.codecool.virtualpub.ui.Display;
+import com.codecool.virtualpub.ui.Input;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 
 public class Bartender {
     private String name;
     private final Random random;
+    private Input input = new Input();
 
     public Bartender(String playerName) {
         this.name = playerName;
         this.random = new Random();
     }
     
     public Drink getDrink(List<Drink> drinks) {
-        // todo get from player in console
-        while (true) {
-            try{
-                // todo getInput()
-                return drinks.get(random.nextInt(drinks.size()));
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        }
+        String[] drinkNames = drinks.stream().map(Drink::getBrand).toArray(String[]::new);
```

Comment:
Do you know what streams are or how they work? Also there is a toList method on streams you can use if you want to avoud working with Arrays